.build/tar/build_exporters.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# The use of >/dev/null is to supress stdout, but allow stderr to come thorugh
+# This is because installing the tools and building makes a lot of output
```

Comment:
Is that a huge problem though? It might not be that bad to just print it all out?

Btw, how long does this script take to run? It is doing quite a lot

Reply:
It is a lot of output, and lots of the specific exporter build scripts are really verbose and run tests suites on the build and such, which looks like a lot for not that much happening.

takes like 30-40 seconds

Reply:
Oh cool that's a lot faster than I expected

Reply:
Was going off of memory, just re-ran it and took a little less than a minute!